debian/changelog

```diff
@@ -1,3 +1,29 @@
+ubuntu-advantage-tools (27.4~22.04.1) jammy; urgency=medium
+
+  * d/tools.postinst:
+    - hardcode python binary and PYTHONPATH
+    - undo unnecessary log file creation
+  * New upstream release 27.4
```

Comment:
Should we include an entry like:
```
detach: notify contract server on detach
```

Reply:
This is one I am not sure, because most of the time we are already doing that, only if the user runs an attach and immediately detaches we will be hitting that issue.
I think we should instead add the following entry:

```
attach: save machine-id during operation
```

What do you think ?

Reply:
Good point! Yes I like your suggestion better